### ISSUE TITLE
 Support continuous video/gif playing from home to detail page (This requires some work to use a whole view controller as a card cell content from the first page!)

### ISSUE THREAD

**Issue by prajwalsprakash** [aunnnn/AppStoreiOS11InteractiveTransition#3]
Hey @aunnnn 

I opened this thread to have more insight by discussing the idea behind what Apple might have done. And to track the updates if we could figure out the same and implement.

Do u think they are hiding part of the whole detail page in the listing cell?
because, If we observe very carefully, their video/gif from the listing continues even after they are expanded to the full screen (detail page) & on dismissal shrinks back to where it came from with hiding rest of the detail part.

If we could make the whole view controller as a part of the cell, would that be feasible? 
We could think about how can we achieve this one rendering part of view controller into the card cell and on-demand execute the full-screen rendering.

**Comment by aunnnn** [comment]
Yeah, I also think that's how it's actually implemented on the App Store. Thanks for bringing this up! I will surely look more into this if I have time.
